content/applications/sales/crm/acquire_leads/generate_from_website.rst

```diff
@@ -1,78 +1,105 @@
-===========================================================
-Generate leads/opportunities from your website contact page
-===========================================================
+==============================
+Generating leads/opportunities
+==============================
 
-Automating the lead/opportunity generation will considerably improve
-your efficiency. Any visitor using the contact form on your website will
-create a lead/opportunity in the pipeline.
+Automating the lead/opportunity generation will considerably improve your team's efficiency.
+Utilizing email aliases and a contact form in your database can aid in that improvement.
 
-Use the contact us on your website
-==================================
+By default, any email sent to *sales@your_database_domain.ext* will create an opportunity in the
+default sales channel's pipeline associated with that particular email.
 
-You should first go to your website app.
+Configuring email aliases
+=========================
 
-|image0|\ |image1|
+Each sales team can use their own email alias to automatically filter generated
+leads/opportunities into their pipeline. This functionality can be useful when managing several
+sales teams with specific individual business processes. You can find Sales Team configurations
+under :menuselection:`CRM --> Configuration --> Sales Teams`.
 
-With the CRM app installed, you benefit from a ready-to-use contact form
-on your Odoo website that will generate leads/opportunities
-automatically.
-
-.. image:: media/generate_from_website03.png
+.. image:: generate_from_website/sales-teams-config.png
```

Comment:
There is a mismatch between the actual filename `sales-team-config.png` and the reference here in the rst `sales-teams-config.png`

Reply:
Thanks for the catch!